spec/factory_bot/aliases_spec.rb

```diff
@@ -1,29 +1,47 @@
 describe FactoryBot, "aliases" do
-  context "aliases for an attribute" do
-    subject { FactoryBot.aliases_for(:test) }
-    it      { should include(:test) }
-    it      { should include(:test_id) }
+  it "aliases for an attribute should include the original attribute" do
+    subject = FactoryBot.aliases_for(:test)
```

Comment:
What do you think of using a more descriptive name than `subject`? I find naming in specs really helps me understand what the implementation does.

```ruby
    aliases = FactoryBot.aliases_for(:test)

    expect(aliases).to include(:test)
```

Reply:
Thanks @eebs !  I just re-named it to `aliases`.